tchannel/tornado/connection.py

```diff
@@ -122,6 +119,15 @@ def __init__(self, connection, tchannel=None):
 
         self.tchannel = tchannel
 
+        # Map from message ID to inflight requests on the server side.
+        self.incoming_requests = {}
+        # Map from message ID to inflight responses on the client side.
+        # When client sends a request, it will first put ID and a future for
+        # response in the _outstanding map. After it gets response message,
+        # but not a completed response, it will put ID and response obj in the
+        # incoming_response.
+        self.incoming_responses = {}
```

Comment:
Will these point to request and response objects? Which ones? The ones defined in `tchannel.tornado.{request,response}`?